ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -61,6 +62,82 @@ export const styles = reactStyles({
   },
 });
 
+const WelcomeHeader = () => {
+  return <FlexColumn style={{marginLeft: '3%', width: '50%'}}>
```

Comment:
removed a `<FlexRow>` and a `<div>` from the original which weren't doing anything